linkerd/proxy/server-policy/src/meta.rs

```diff
@@ -39,6 +39,10 @@ impl Meta {
             Self::Resource { group, .. } => group,
         }
     }
+
+    pub fn is_audit(&self) -> bool {
+        self.kind() == "default" && self.name() == "audit"
```

Comment:
We can use pattern matching to avoid string comparison here:

```suggestion
        matches!(self, Self::Default { name } if name == "audit")
```